app/lib/frontend/backend.dart

```diff
@@ -269,6 +269,35 @@ class Backend {
       }
     }
   }
+
+  /// Updates flags on [package].
```

Comment:
```dart
/// Updates flags on [package], as the user of the current request.
```

or is that implicit?

Reply:
At other methods of the `Backend` class we already use the current user implicitly (e.g. upload, uploader add/remove), I think we could assume that for future methods we'd have the same implicit assumption. Or would you like to have it differently?

Reply:
nvm :)